src/components/Label/Label.tsx

```diff
@@ -65,12 +73,14 @@ class Label extends UIComponent<ReactProps<LabelProps>, any> {
     image: customPropTypes.itemShorthand,
     imagePosition: PropTypes.oneOf(['start', 'end']),
     fluid: PropTypes.bool,
+    rtlAttributes: PropTypes.func,
   }
 
   static defaultProps = {
     as: 'span',
     imagePosition: 'start',
     iconPosition: 'end',
+    rtlAttributes: childrenDependentRtlAttributes,
```

Comment:
just to clarify for myslef - could you, please, suggest why it cannot be used as a default RTL attributes provider?

Reply:
or, I see, nvm